src/palindrom.js

```diff
@@ -487,12 +487,20 @@ var Palindrom = (function() {
     reconnectionFn(function bootstrap(json) {
       palindrom.reconnector.stopReconnecting();
 
-      palindrom.queue.reset(palindrom.obj, json);
-
       if (palindrom.debug) {
         palindrom.remoteObj = JSON.parse(JSON.stringify(json));
       }
 
+      if(!palindrom.isObjectProxified) { // first time the state is being set
```

Comment:
Ah, name Ok, maybe this will not be very expensive, but switching it on just to switch it ofafterwards and set and reset the callbacks, is a redundant code, that complicates the flow.

Maybe we could split `observe` to `proxify` and `observeStart`?

Is it possible to call `observe` before recconect code? or without `obj` parameter if `!this.isObjectProxified` - that would result in Proxifying `undefined`. Such flow of `unobserve` definition looks like a code smell to me